pages/apis/rest_api/pipelines.md.erb

```diff
@@ -687,19 +687,6 @@ Additional properties available for GitHub:
         What type of event to trigger builds on. <code>Code</code> will create builds when code is pushed to GitHub. <code>Deployment</code> will create builds when a deployment is created with the <a href="https://developer.github.com/v3/repos/deployments/">GitHub Deployments API</a>. <code>Fork</code> will create builds when the GitHub repository is forked. <code>None</code> will not create any builds based on GitHub activity.
         <p class="Docs__api-param-eg"><em>Values:</em> <code>code</code>, <code>deployment</code>, <code>fork</code>, <code>none</code></p></td>
     </tr>
```

Comment:
```suggestion
    </tr>
    <tr>
      <th><code>filter_enabled</code></th>
      <td>
        Whether filter conditions are being used for this step.
        <p class="Docs__api-param-eg"><em>Values:</em> <code>true</code>, <code>false</code></p></td>
    </tr>
    <tr>
      <th><code>filter_condition</code></th>
      <td>
        The conditions under which this step will run. See the <a href="/docs/pipelines/conditionals">Using Conditionals</a> guide for more information.
        <p class="Docs__api-param-eg"><em>Example:</em> <code>"build.message != 'skip'"</code></p>
      </td>
    </tr>        
```

Reply:
@vgrigoruk let's put these back, I'll open a separate PR when I hear the situation resolved, and we can merge your fixes to `publish_blocked_as_pending`